CIP-0035/README.md

```diff
@@ -0,0 +1,38 @@
+---
+CIP: 35
+Title: Daedalus URI Scheme
+Authors: Daniel Main <daniel.main@iohk.io>
+Comments-URI:
+- https://github.com/cardano-foundation/CIPs/pull/130
```

Comment:
great that you have the last PR discussion in there, but can you also add this one?
`- https://github.com/cardano-foundation/CIPs/pull/234`